Mathlib/Combinatorics/Quiver/Path.lean

```diff
@@ -153,8 +153,8 @@ theorem comp_inj {p₁ p₂ : Path a b} {q₁ q₂ : Path b c} (hq : q₁.length
 
 theorem comp_inj' {p₁ p₂ : Path a b} {q₁ q₂ : Path b c} (h : p₁.length = p₂.length) :
     p₁.comp q₁ = p₂.comp q₂ ↔ p₁ = p₂ ∧ q₁ = q₂ :=
-  ⟨fun h_eq => (comp_inj <| Nat.add_left_cancel <| by simpa [h] using congr_arg length h_eq).1 h_eq,
-    by
+  ⟨fun h_eq => (comp_inj <| Nat.add_left_cancel <| by
+    simpa [h] using congr_arg length h_eq).1 h_eq, by
     rintro ⟨rfl, rfl⟩
     rfl⟩
```

Comment:
The indentation here is a bit misleading. I think it is better to break before the by, and just do something like
```suggestion
  ⟨fun h_eq => (comp_inj <| Nat.add_left_cancel <| by simpa [h] using congr_arg length h_eq).1 h_eq,
   by rintro ⟨rfl, rfl⟩; rfl⟩
```
Alternatively, if both tactic blocks are large, I would generally just use a single `by` block that starts with a `refine` producing two subgoals and then use `·` for each goal. That is a lot easier to format nicely.